src/source/reference/get-campaign.rst

```diff
@@ -23,4 +23,4 @@ GET /campaign v5
 Example
 """""""
 
-.. include:: ../examples/get-campaign-v5-example.rst
\ No newline at end of file
+.. include:: ../examples/get-campaign-v5-example.rst
```

Comment:
should also rename this example file then

Reply:
Done.